### ISSUE TITLE
Running Pentoo on a dedicated hard drive

### ISSUE THREAD

**Issue by john44066** [pentoo/pentoo-overlay#268]
I have been trying to figure out how to run Pentoo off of a dedicated hard drive in my system so I don't have to use the DVD. I installed a new 2 Tb hard drive specifically to run Linux and GNU Radio Companion but all my research has come up empty on how to do this. Could someone point me in the right direction to run Pentoo directly off of a hard drive? That way my system would be setup as a dual boot where I would physically have to setup to boot off of either hard drive via the F12 key for boot selection on my system.

I like the setup of Pentoo and its GUI. It is very nice to use and would prefer to use that over any other install unless someone has a better suggestion.

Regards,

John

**Comment by brianmwaters** [comment]
This is definitely not the right forum for this. Maybe Pentoo has a mailing list? I know there is a channel on Freenode.

Anyway, I for one have never used the Pentoo installer or live CD, but I have added the Pentoo overlay on top of Gentoo using Layman, and enabled the Pentoo profile. That's one approach if you're game to go down the Gentoo rabbit hole.

**Comment by john44066** [comment]
Well, I am new to all this. The Pentoo's Forum has been down for maintenance for quite some time now. It states: "Maintenance in progress... Please contact us through irc in the meantime irc://irc.freenode.net/#pentoo" so I tried to gain access to irc.freenode  and I managed to gain access but that is also down as well as far as I could determine. So this blog looked like a good place to start as I have no other alternative that I could find.

I'd prefer to keep with Pentoo for the time being since that is what is recommended by mossmann for HackRF which is my goal in all this. Maybe someone else will chime in with an idea.

Thank you for your response.

John

**Comment by blshkv** [comment]
Pentoo installer does not support dual boot at this moment.
The best approach would be to follow Gentoo documentation and install it manually as an overlay.

**Comment by john44066** [comment]
Okay, thank you for that. I did see the data on Gentoo and the fact that it can be overlaid but there is writing indicating that people have had problems doing that. I hate to add more stuff only to get deeper into this when all I really need to do is use GNU Radio Companion with my HackRF ONE product that is coming in the mail soon. I would really like to run GNU Radio on a Windows machine since I do not have any Apple gear and I have no experience with Apple products. I once bought an Apple MacBook Pro laptop and had it two days and returned it since I could not even get it past the setup. Best Buy showed me how to do it and it was very convoluted. Also as a Windows user switching over to Apple Macs is painful based on my short experience with it.

**Comment by brianmwaters** [comment]
Trying to install and use GNU Radio on Windows is... a nightmare, at best.

Maybe you'd be better off just running Ubuntu, and installing the version of GNU Radio that comes with that? Its possible it will be a few minor versions out of date but you should be fine.

**Comment by john44066** [comment]
Can I install Ubuntu on my dedicated hard drive that I just installed for my HackRF ONE work and then select to boot from that drive on startup of my computer?

**Comment by john44066** [comment]
Do I use the LTS (Long term support) version 16.04.3 or the non LTS version 17.10.1?

**Comment by john44066** [comment]
Okay, I will look into Umbutu. Thanks for the tip.

John

**Comment by blshkv** [comment]
funny enough, you don't need to run a Pentoo since we pushed gnuradio to Gentoo.
Just install gentoo and emerge gnuradio.

**Comment by john44066** [comment]
Ok, thank you for that information.

**Comment by john44066** [comment]
So I tried the live version of GNU Radio with Ubuntu via an ISO on a DVD and it works very, very good. I have audio, internet, and good ability to load external applications such as VLC Media Player (used to test the audio). This live version cannot be put on a hard drive. I did find instructions in setting up a side by side install of my current Windows 7 and Ubuntu so that will be my next effort. Thank you brianmwaters for the suggestion. I do appreciate that very much.

John
